### DISCUSSION TITLE
Make G-Helper an Installed App

### DISCUSSION THREAD

**Discussion by mapm1998** [seerge/g-helper#1239] [Ideas]
i don't know if this topic has already been discussed (by my search, it hasn't), but why not make G-Helper an installed app, with a dedicated folder where the updates go automatically?
This could make the app more simpler with the updates, I think (the app, even with the auto-update, it doesn't do that).
If this has already been discussed, my apology.

**Comment by seerge** [comment]
@mapm1998 hello, app is currently by design "portable" (i.e. single exe that you can run from anywhere)

Auto-update actually replaces old version with a new one, so it works fine
